src/screens/accountCreationScreen.js

```diff
@@ -1,15 +1,35 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {
   SafeAreaView,
   StyleSheet,
   TouchableWithoutFeedback,
+  TouchableHighlight,
   View,
   Text,
   TextInput,
   Keyboard,
 } from 'react-native';
+import {useNavigation} from 'react-navigation-hooks';
+import {useSession} from '../firebase/auth';
+import {initializeUserInDatabase} from '../firebase/firestore';
+import storage from '@react-native-firebase/storage';
 
 const AccountCreationScreen = () => {
+  const session = useSession();
+  const {navigate} = useNavigation();
+  const [userInfo, changeUserInfo] = useState({
```

Comment:
By convention, naming would be 
```suggestion
  const [userInfo, setUserInfo] = useState({
```
This makes it more clear that values passed to this function overwrite current state, instead of appending to it or merging with it.